tests/ci/run_check.py

```diff
@@ -189,16 +175,32 @@ def main():
             pr_info,
         )
         sys.exit(1)
-    else:
-        print("::notice ::Can run")
+
+    if FEATURE_LABEL in pr_info.labels and not pr_info.has_changes_in_documentation():
+        print(
+            f"The '{FEATURE_LABEL}' in the labels, "
+            "but there's no changed documentation"
+        )
+        PR_CHECK = "PR Check"
         post_commit_status(
             commit,
-            "pending",
-            ci_report_url,
-            description,
-            CI_STATUS_NAME,
+            FAILURE,
+            "",
+            f"expect adding docs for {FEATURE_LABEL}",
+            PR_CHECK,
             pr_info,
         )
+        update_mergeable_check(commit, pr_info, PR_CHECK)
```

Comment:
This still should be `if : else` with line 164. We don't need to update it a few times, only once.

And if we use `PR_CHECK` exclusively once, I'd stick to docs.

I get the changes regarding `exit(0)`. It feels that it's ok to continue the workflow now.  
The other changes around `update_mergeable_check` are very useful.